test-helpers/src/lib.rs

```diff
@@ -38,16 +38,73 @@ pub fn run_command(command: &str, args: &[&str]) -> Result<String> {
     }
 }
 
+fn command_not_found_error_message(command: &str, args: &[&str]) -> String {
```

Comment:
Another nit:
Would probably make sense to return an anyhow::Error here instead of a string.
But I wont block on this either